__tests__/index.spec.js

```diff
@@ -517,7 +518,23 @@ describe('one-app-dev-cdn', () => {
       expect(moduleResponse.status).toBe(200);
       expect(moduleResponse.header['content-type']).toMatch(/^application\/javascript/);
       expect(moduleResponse.text).toBe('console.log("a");');
-      expect(got.mock.calls).toMatchSnapshot('module response');
+      expect(got.mock.calls).toEqual([
+        [
+          'https://example.com/module-map.json',
+          {
+            agent: expect.any(ProxyAgent),
+          },
+        ],
+        [
+          'https://example.com//cdn/module-b/1.0.0/module-b.node.js',
```

Comment:
where is the `//` coming from ?
```suggestion
          'https://example.com/cdn/module-b/1.0.0/module-b.node.js',
```

Reply:
this should be looked at and fixed in a followup pr